scheduler/cloudformation/cloudformation.go

```diff
@@ -167,6 +167,17 @@ type SubmitOptions struct {
 	// When true, does not make any changes to DNS. This is only used when
 	// migrating to this scheduler
 	NoDNS bool
+
+	// When provided, will generate a new RestartKey for the target.
+	RestartTarget string
+}
+
+func (s *Scheduler) Restart(ctx context.Context, app *scheduler.App, p string) error {
```

Comment:
I think the endgoal, once we can remove the old ECS scheduler, would be to change this method signature to `Restart(context.Context, app string, process string) error`, since we won't need to submit a new template to cloudformation, and can just update a parameter.

For that reason, I think we should still make this just update parameters, instead of doing a full Submit and generating a new template.